cirq-core/cirq/__init__.py

```diff
@@ -330,6 +330,7 @@
 
 from cirq.qis import (
     bloch_vector_from_state_vector,
+    choi,
```

Comment:
Should this be a more specific name here to avoid future naming conflict? `operation_to_choi` or something?

Reply:
Good point, it does appear we might have a protocol named `choi` in near future.